src/index.ts

```diff
@@ -0,0 +1,88 @@
+import crypto from 'crypto';
+import { Request, RequestHandler, NextFunction } from 'express';
+
+import stringToBuffer from './stringToBuffer';
+import validateArguments from './validateArguments';
+import AuthError from './errors';
+
+export type Options = {
+    algorithm: string;
+    identifier: string;
+    header: string;
+    maxInterval: number;
+    minInterval: number;
+}
```

Comment:
Tiny thing and somewhat debated, but the typescript docs advise to use `interface` until you need the features of `type`. In this case it would make no difference, but the differences are something to be aware of if asked.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces